### ISSUE TITLE
Getting warning from numpy when creating histogram

### ISSUE THREAD

**Issue by sean-morris** [data-8/datascience#551]
I am using datascience 0.17.5 with matplotlib 3.5.2

![image](https://user-images.githubusercontent.com/1514995/184213943-5e34c656-14e9-4b39-85a4-27cad4df6757.png)

This seems to talk about a workaround:
https://stackoverflow.com/questions/63723514/userwarning-fixedformatter-should-only-be-used-together-with-fixedlocator

**Comment by SakshiSingh25** [comment]
Hi, I am interested in working on this issue. Kindly, guide me in the further process.
Thank you.

**Comment by ToshiNandanReddy** [comment]
Hello, I am interested in working on this issue.  Could you please guide me with further steps.
Thank you.

**Comment by helloaadii** [comment]
Hello, I am interested in working on this issue. Could you please guide me with further steps.
Thank you.

**Comment by davidwagner** [comment]
I'm afraid we don't have resources to guide you on working on this.  Sorry.

**Comment by adnanhemani** [comment]
Can confirm that I can reproduce this on datascience 0.17.6 and matplotlib 3.5.3. Minimalistic example code:
```
from datascience import *
import matplotlib.pyplot as plt
%matplotlib inline
plt.style.use('fivethirtyeight')
from bokeh.sampledata.autompg import autompg as df
t = Table.from_df(df)
t["mpg_re"] = t["mpg"] * 100000000
t.hist('mpg_re')
```

Investigating fix options now.

**Comment by adnanhemani** [comment]
Closed by #574. Thanks for the bug report @sean-morris!
